### ISSUE TITLE
Surveyor for Ruby 2.0.0?

### ISSUE THREAD

**Issue by edwardsf** [NUARIG/surveyor#447]
I'm interested to find out if surveyor will be available for Ruby 2.0.0 soon?

**Comment by rsutphin** [comment]
We haven't done any testing on Ruby 2.0, but based on my understanding of Ruby 2.0 I wouldn't expect anything to break. If you can provide examples of things that don't work, that would be helpful.

**Comment by edwardsf** [comment]
Installation itself didn't work, so I just considered it no available for Ruby 2.0.0 installations. Is there something I'm missing?

**Comment by bigtunacan** [comment]
We were experiencing some compatibility issues with Ruby 2.0 on one of our projects, but it was unclear at first if it was an issue with Surveyor, or with extensions/changes we have made to our fork.  So to narrow it down I created a clean project on surveyor with the kitchen sink surveyor against Ruby 2.0.0p195.  Surveyor appears to work just fine with Ruby 2.0. I would say this issue can be closed.
